public/scripts/main/services/untitled

```diff
@@ -0,0 +1,52 @@
+this.getUsersPoints = function (cb) {
```

Comment:
please do not load a file with no file type into master... what is its purpose? this is not an executable or readable file since it has no type...